charts/airflow/values.yaml

```diff
@@ -412,6 +415,10 @@ airflow:
     ##
     securityContext: {}
 
+    ## the priorityClassName for the db-migrations Pods
+    ##
+    priorityClassName: ""
```

Comment:
Let's add a link for people who don't know what `priorityClassName` is:

```yaml
  ## the priorityClassName for the db-migrations Pods
  ## - docs for priorityClassName:
  ##   https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
  ##
  priorityClassName: ""
```

__NOTE:__ we will also need to add it to all the other `priorityClassName` docstrings in this values file